source/parser.hera

```diff
@@ -6708,18 +6708,19 @@ EnumBlock
 
 NestedEnumProperties
   PushIndent NestedEnumProperty*:props PopIndent ->
-    if (!props.length) return $skip
```

Comment:
I think this should have stayed. The point is to skip if nothing was matched, not if there end up being no properties (though these are equivalent).